aspnet/mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/adding-a-new-field.md

```diff
@@ -2,7 +2,7 @@
 uid: mvc/overview/older-versions/getting-started-with-aspnet-mvc3/cs/adding-a-new-field
 title: "Adding a New Field to the Movie Model and Table (C#) | Microsoft Docs"
 author: Rick-Anderson
-description: "This tutorial will teach you the basics of building an ASP.NET MVC Web application using Microsoft Visual Web Developer 2010 Express Service Pack 1, which is..."
+description: "This tutorial will teach you the basics of building an ASP.NET MVC Web application using Microsoft Visual Web Developer 2010 Express Service Pack 1. You will learn how to add a rating property to the movie model, manage model and database schema differences, and automatically re-create the database on model changes."
```

Comment:
"Learn how to add a rating property to the movie model, manage model and database schema differences, and automatically re-create the database on model changes."